### ISSUE TITLE
Cancel save window with ESC

### ISSUE THREAD

**Issue by llrs** [rstudio/rstudio#4455]
From time to time when I start writing a new file I tend to press CTRL+S to save the file (of pure habit). Then I realize it is not ready (or I don't know a name of the file yet) and press ESC to exit the window. However, ESC doesn't cancel the action. 

Could it be added that ESC cancel/closes the dialogue to save the file?

I couldn't see a similar request and I hope it is fairly easy to be added on the next version of RStudio.

**Comment by kevinushey** [comment]
Please take the time to fill out the template, so we can better understand the environment in which this bug was seen.

### System details

    RStudio Edition : 
    RStudio Version : 
    OS Version      : 
    R Version       : 

### Steps to reproduce the problem

### Describe the problem in detail

### Describe the behavior you expected

**Comment by kevinushey** [comment]
This should normally work. What version of RStudio (desktop, server) and OS are you working with?

**Comment by gtritchie** [comment]
Throwing the Accessibility label on the issue, in case we find a repro.

**Comment by llrs** [comment]
Ok, checking it in my personal computer (Ubuntu 18.10 and RStudio desktop 1.1.463) it works but the same version of RStudio (desktop) on the 16.04 version of Ubuntu at work doesn't work. Both OS are 64 bytes

**Comment by kevinushey** [comment]
Have you tried the [preview release](https://www.rstudio.com/products/rstudio/download/preview/)?

**Comment by llrs** [comment]
Not yet, I'll do it tomorrow

**Comment by llrs** [comment]
Ok, I can't reproduce it now at work, but I certainly have seen it happening (as well as other unexpected errors in RStudio...)
